Eth-Global-Guide.md

```diff
@@ -0,0 +1,32 @@
+# Intro to Zetachain
+
+## Introduction
+
+- What is ZetaChain?
+- Why build dApps on Zeta ? (Key features) 
+- What is ZETA token ?
+
+## Build with Zeta
+- Start building dapps ( Refer to the first half -  https://www.zetachain.com/docs/developers/overview/) 
+- Using Zetachain Smart Contracts (conceptual explanation)
+> - Omnichain Contracts
+> - Cross Chain Messaging (explanation 
+- Sending and Depositing Tokens
+- Smart Contract Template
+
+## Smart Contracts
+- Create a simple Omnichain Contract
+- Create a simple Cross Chain Messaging Contract
+
+
+## FullStack tutorial 
+- Bitcoin Frontend (Omnichain)
+- CCM
+
+## Developer Resources
+- Installing Zetachain CLI
```

Comment:
If you mean installing [`zetacored`](https://github.com/zeta-chain/node), then I think it's a bit out of scope for regular developers. You don't need a ZetaChain node running locally to develop omnichain contracts.